src/extension/providers/nostr/index.ts

```diff
@@ -52,16 +51,15 @@ export default class NostrProvider {
 
   async on(...args: Parameters<EventEmitter["on"]>) {
     await this.enable();
-    return this._eventEmitter.on(...args);
+    this._eventEmitter.on(...args);
   }
   async emit(...args: Parameters<EventEmitter["emit"]>) {
-    await this.enable();
-    return this._eventEmitter.emit(...args);
+    this._eventEmitter.emit(...args);
```

Comment:
this function does not need to be async any more, does it?

Reply:
aah thanks!